features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,20 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  within("#session") do
+    container = first('.container')
+    container.should_not == nil
+  end
 end
 
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  within("#session") do
+    fill_in 'username', :with => 'test'
```

Comment:
Started using rubocop right after and it gave me a heads up on the new syntax haha